### ISSUE TITLE
Language code is not correct for Chinese

### ISSUE THREAD

**Issue by vinzao** [wagtail/wagtail#3668]
#### Please refer to the link below.

[https://code.djangoproject.com/ticket/18419](https://code.djangoproject.com/ticket/18419)

> This is because Traditional Chinese is not used only by Taiwan (tw) but also Honk Kong (hk), and Simplified Chinese is not only used in China (cn) but also Singapore (sg) and Malaysia. A new standard is to use "zh_Hant" for Traditional Chinese and "zh_Hans" for Simplified Chinese to eliminate the confusion of the content is not just for people lived in China or Taiwan.

**Comment by yihuang** [comment]
Maybe it's sufficient to create a symlink in locale directory from ``zh_Hans`` to ``zh_CN``, does it works with transifex tool?

**Comment by gasman** [comment]
The `zh_CN` and `zh_Hant` translations have now reached 100% completion on Transifex (a huge thank you to everyone involved!), so I've now removed the incomplete `zh_TW` translation and (in d5bc3c92d22672f06f2aeab1e1731ca4f4fcb383) updated the Transifex configuration to download these to the new locations `zh_Hans` and `zh_Hant`. Hopefully this solves the issue.

I've tested this locally and it seems to work - characters are changing when I switch between the locales - but I don't know Chinese, so I'd encourage other people to test this, and let me know on this ticket if anything isn't right. (There will be a release candidate of Wagtail 2.5 published shortly.)
